include/net/arp.h

```diff
@@ -17,6 +17,12 @@
 
 #include <net/ethernet.h>
 
+/**
+ * @brief Address resolution (ARP) library
+ * @defgroup arp ARP Library
```

Comment:
defgroup is defined now (yea!) but there are no other doxygen comments in this file.  (I notice though it says in the comments, "This is not to be included by the application." so maybe that's not a problem.)